docker-compose.override.yml

```diff
@@ -26,7 +26,34 @@ services:
       "--app", "rgd.celery",
       "worker",
       "--loglevel", "INFO",
-      "--without-heartbeat"
+      "--without-heartbeat",
+      "-Q default"
```

Comment:
You could use `-X` / `--exclude-queues` instead: https://docs.celeryproject.org/en/stable/reference/cli.html#cmdoption-celery-worker-X